iscsi/initiator.go

```diff
@@ -119,21 +119,22 @@ func LogoutTarget(ip, target string, ne *util.NamespaceExecutor) error {
 	return nil
 }
 
-func GetDevice(ip, target string, lun int, ne *util.NamespaceExecutor) (string, error) {
+func GetDevice(ip, target string, lun int, ne *util.NamespaceExecutor) (string, string, error) {
```

Comment:
It's a bit weird to return multiple strings. And it's better if we can return major/minor number directly instead of put them in a string.

How about this:
```
type KernelDevice struct {
    Name string
    Major int
    Minor int
}
```

Can be defined in the `/util`

And calling `major:minor` "driver" seems not accurate.

Better return a pointer to the struct here for this function, so when one encountered an error, `return nil, err`